### ISSUE TITLE
For markdown solution is pre-fed in the code 

### ISSUE THREAD

**Issue by aashishvanand** [exercism/python#2862]
For [Markdown](https://exercism.org/tracks/python/exercises/markdown) exercise the problem is pre-fed. 
Fix: remove the solution by modifying existing code

**Comment by BethanyG** [comment]
Hi @aashishvanand 👋🏽 

Thanks for filing an issue!  Two points:

1.  This is a **refactoring** exercise, as is stated in the exercise instructions (_emphasis mine_):
> _Refactor a Markdown parser._
>
>_The markdown exercise is a refactoring exercise. There is code that parses a given string with Markdown syntax and returns the associated HTML for that string. Even though this code is confusingly written and hard to follow, somehow it works and all the tests are passing! **`Your challenge is to re-write this code to make it easier to read and maintain while still making sure that all the tests keep passing.`**_
>
>_It would be helpful if you made notes of what you did in your refactoring in comments so reviewers can see that, but it isn't strictly necessary. The most important thing is to make the code better!_

2.  This is a **practice exercise**.  Practice exercise problem descriptions are pulled from the common [problem specifications](https://github.com/exercism/problem-specifications) repo.  Substantial changes to them are proposed there, so that they can be discussed cross-track by maintainers.  In future, if there is an issue with the way a practice exercise is presented (_as opposed to an issue pertaining to the Python programming language_), you should file an issue there. 

Given that the code is necessary in the stub file (_so that it can be refactored_) and not "the answer" to this exercise, I am going to close this issue as **`won't fix`**.
